openid-sharedsignals-framework-1_0.md

```diff
@@ -315,6 +315,9 @@ All members within a Complex Subject MUST represent attributes of the same
 Subject Principal. As a whole, the Complex Subject MUST refer to exactly one
 Subject Principal.
 
+For details about how to interpret unspecified claims in a Complex Subject as wildcards,
```

Comment:
```suggestion
For details about how to interpret unspecified claims in a Complex Subject,
```

I feel that the concept of a "wildcard" has not been defined, but it is implicitly assumed in this PR. Let's try to avoid it, without a clear definition.

Reply:
My concern here is that people might search the spec for this kind of behavior using the term "wildcard". How would you feel about leaving this sentence in but changing it in the others?

Reply:
Discussed in call. All in agreement.